mw-webapp/src/service/WayService.ts

```diff
@@ -80,6 +85,39 @@ export class WayService {
     return validatedWaysDTO;
   }
 
+  /**
+   * Get WaysDTO amount
+   */
+  public static async getWaysDTOAmount(): Promise<number> {
+    const waysRef = collection(db, PATH_TO_WAYS_COLLECTION);
+    const snapshot = await getCountFromServer(waysRef);
+    const waysAmount = snapshot.data().count;
+
+    const readsAmount = Math.ceil(waysAmount / AMOUNT_DOCS_FOR_COUNT_READS);
+
+    logToConsole(`WayService:getWaysDTOAmount: ${readsAmount} ${RequestOperations.READ} operations`);
+
+    return waysAmount;
+  }
+
+  /**
+   * Get WaysDTO
+   */
+  public static async getWaysDTOPart(date: Date): Promise<WayDTO[]> {
```

Comment:
what is part? why do we need date if we don't have any filters?

Reply:
Because pagiantion works only for ordered list  of elems/. Ways we orders by date

Reply:
related to pagination somehow